classes/form/client.php

```diff
@@ -57,6 +57,8 @@ public function definition() {
         $mform->addRule('primarydomain', get_string('maximumchars', '', 255), 'maxlength', 255, 'server');
         $mform->addHelpButton('primarydomain', 'domain', 'mod_scormremote');
 
+        $mform->addElement('date_time_selector', 'expiry', get_string('expiry', 'mod_scormremote'), array('optional' => true));
```

Comment:
One last thing - this string doesn't exist.

Reply:
Missed that resolving the cherry pick... sorted now. Thanks for your patience!